### PR TITLE
SEQNG-654: Support CRFOLLOW

### PR DESCRIPTION
**Pull request by cquiroz**
This adds support for the CRFOLLOW keyword in Gpi and also fixes reading the corresponding epics channels from the TCS

### CODE REVIEW

File: modules/seqexec/server/src/main/resources/Tcs.xml [hunk 1]
```diff
@@ -1006,6 +1006,16 @@
     </Status>
     <Status name="tcsstate">
         <top>tcs</top>
+        <attribute name="crfollow">
+            <channel>crFollow</channel>
```
Comment (jluhrs) [inline modules/seqexec/server/src/main/resources/Tcs.xml]: crFollow.VAR, to be precise.

File: modules/seqexec/server/src/main/resources/Tcs.xml [hunk 2]
```diff
@@ -1006,6 +1006,16 @@
...
+            <type>INTEGER</type>
+            <description>CASS Rotator Follow</description>
+        </attribute>
+        <attribute name="parangle">
+            <channel>parAngle</channel>
```
Comment (jluhrs) [inline modules/seqexec/server/src/main/resources/Tcs.xml]: Its "sad:parAngle.VAL". Why this has the sad: prefix and the crfollow doesn't? No idea.

File: modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala [hunk 3]
```diff
@@ -104,6 +104,28 @@ object Tcs {
...
+        case 0  => Off.some
+        case 1  => On.some
+        case _  => none
+      } {
+        case Off => 0
+        case On  => 1
+      }
+
+  }
+
```
Comment (jluhrs) [inline modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala]: I am not sure this class is the right place for this piece of code.

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala]: Do you mean the whole CRFollow or just the prism. The Prism is used to decode the epics values

File: modules/seqexec/server/src/main/resources/Tcs.xml (hunk 2 above)
Comment (cquiroz) [reply modules/seqexec/server/src/main/resources/Tcs.xml]: [software@sbfgpidev1 ~]$ caget tcs:sad:parAngle.VAL
tcs:sad:parAngle.VAL           -41.472

File: modules/seqexec/server/src/main/resources/Tcs.xml (hunk 1 above)
Comment (cquiroz) [reply modules/seqexec/server/src/main/resources/Tcs.xml]: [software@sbfgpidev1 ~]$ caget tcs:crFollow.VAL
tcs:crFollow.VAL               0

File: modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala [hunk 4]
```diff
@@ -104,6 +104,28 @@ object Tcs {
...
+  object CRFollow {
+    case object On extends CRFollow
+    case object Off extends CRFollow
+
+    def keywordValue(cr: CRFollow): String = cr match {
+      case On  => "yes"
+      case Off => "no"
+    }
+
+    def fromInt: Prism[Int, CRFollow] =
```
Comment (tpolecat) [inline modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala]: Can you add a law test for this prism?

**Pull request merged**